src/Twig/FrontendMenuExtension.php

```diff
@@ -58,8 +58,11 @@ public function renderMenu(Environment $twig, ?string $name = null, string $temp
 
     public function isCurrent($item): bool
     {
+        $currentRequest = $this->requestStack->getCurrentRequest();
+        $currentLocale = $currentRequest->getLocale();
         $uri = $item['uri'] ?? '';
+        $currentUrl = str_replace('/' . $currentLocale, '', $currentRequest->getPathInfo());
```

Comment:
I think this is a good fix in general, just not sure if this won't cause an edgecase.. Would it be better to include the the slash _after_ the locale as well? 

```suggestion
        $currentUrl = str_replace('/' . $currentLocale, '/', '/', $currentRequest->getPathInfo());
```

Reply:
You are right